docs/source/user_guide/automl.ipynb

```diff
@@ -645,17 +1598,1393 @@
+    "\n",
+    "Note: the cell demonstrating process-level parallelism is commented out due to incompatibility with our ReadTheDocs build.  It can be run successfully locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-07-27T18:56:20.143754Z",
+     "start_time": "2021-07-27T18:56:15.172329Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Using default limit of max_batches=1.\n",
+      "\n",
+      "Generating pipelines to search over...\n",
+      "2 pipelines ready for search.\n",
+      "\n",
+      "*****************************\n",
+      "* Beginning pipeline search *\n",
+      "*****************************\n",
+      "\n",
+      "Optimizing for Log Loss Binary. \n",
+      "Lower score is better.\n",
+      "\n",
+      "Using CFEngine to train and score pipelines.\n",
+      "Searching up to 1 batches for a total of 3 pipelines. \n",
+      "Allowed model families: linear_model\n",
+      "\n",
+      "Evaluating Baseline Pipeline: Mode Baseline Binary Classification Pipeline\n",
+      "Mode Baseline Binary Classification Pipeline:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 12.868\n",
+      "\n",
+      "*****************************\n",
+      "* Evaluating Batch Number 1 *\n",
+      "*****************************\n",
+      "\n",
+      "Elastic Net Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "Logistic Regression Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "\tHigh coefficient of variation (cv >= 0.2) within cross validation scores.\n",
+      "\tLogistic Regression Classifier w/ Imputer + Standard Scaler may not perform as estimated on unseen data.\n",
+      "\n",
+      "Search finished after 00:03            \n",
+      "Best pipeline: Logistic Regression Classifier w/ Imputer + Standard Scaler\n",
+      "Best pipeline Log Loss Binary: 0.076807\n"
+     ]
+    }
+   ],
+   "source": [
+    "from concurrent.futures import ThreadPoolExecutor\n",
+    "\n",
+    "from evalml.automl.engine import CFEngine\n",
+    "from evalml.automl.engine.cf_engine import CFClient\n",
+    "\n",
+    "# Use thread-level paralellism\n",
+    "threaded_cf_engine = CFEngine(CFClient(ThreadPoolExecutor()))\n",
+    "\n",
+    "automl_cf_threaded = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                      problem_type=\"binary\",\n",
+    "                      allowed_model_families=[ModelFamily.LINEAR_MODEL],\n",
+    "                      engine = threaded_cf_engine)\n",
```

Comment:
Hypernit: Can we fix the spacing here? This is what I see
![image](https://user-images.githubusercontent.com/22552445/127228505-22ba4bb2-8536-4449-b568-b9159fbfa48b.png)